### PR TITLE
Profile Examples: CUDA8 @hypnos-hzdr

### PR DESCRIPTION
**Pull request by ax3l**
Module update for Hypnos (HZDR): CUDA 8.0, C++11 compile chain with up-to-date file I/O plugins.

Currently being installed:
- [x] wait for official ok from admin to test them
- [x] Splash 1.6.0 module
- [x] ADIOS 1.10.0 module
- [x] do minimal RT test

### CODE REVIEW

**Comment by ax3l** [comment]
ping @psychocoderHPC @PrometheusPi

File: src/picongpu/submit/hypnos-hzdr/picongpu.profile.example [hunk 1]
```diff
@@ -8,22 +8,22 @@ then
...
-        module load cuda/6.5
-        module load openmpi/1.8.4.kepler
+        module load boost/1.62.0
+        module load cuda/8.0
+        module load openmpi/1.8.6.kepler.cuda80
 
         # Plugins (optional)
         module load pngwriter/0.5.6
-        module load hdf5-parallel/1.8.14 libsplash/1.6.0
+        module load hdf5-parallel/1.8.15 libsplash/1.6.0
```
Comment (ax3l) [inline src/picongpu/submit/hypnos-hzdr/picongpu.profile.example]: still setting them up - I just tried to load them now.

let us wait until I get the mail they are officially there (updated description)

File: src/picongpu/submit/hypnos-hzdr/picongpu.profile.example [hunk 2]
```diff
@@ -8,22 +8,22 @@ then
...
 
         # either use libSplash or ADIOS for file I/O
-        #module load libmxml/2.8 adios/1.10.0
```
Comment (psychocoderHPC) [inline src/picongpu/submit/hypnos-hzdr/picongpu.profile.example]: Note: you removed **libmxml**

Comment (ax3l) [inline src/picongpu/submit/hypnos-hzdr/picongpu.profile.example]: yep, not required in adios 1.10+ any more :)

File: src/picongpu/submit/hypnos-hzdr/picongpu.profile.example (hunk 1 above)
Comment (ax3l) [inline src/picongpu/submit/hypnos-hzdr/picongpu.profile.example]: ok they are announced now, I am testing one...

Comment (ax3l) [inline src/picongpu/submit/hypnos-hzdr/picongpu.profile.example]: works :)

**Pull request merged**